### PR TITLE
feat: :sparkles: Initial commit

### PR DESCRIPTION
**Pull request by ntokozo-nhlangulela**
Added cards to the code.

### CODE REVIEW

File: src/components/mangaCard.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import Typography from "@mui/material/Typography";
+import { useEffect, useRef, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import MangaApis from "../agent/api";
+import MangaList from "../models/mangaList";
+import appPaths from "../shared/paths";
+
+//coverArt: item.relationships.attributes.fileName,
+
+const T = () => {
```
Comment (CharelMarais) [inline src/components/mangaCard.tsx:16]: The name T does not explain what the component does.

File: src/components/mangaCard.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+              fileName: name,
+            },
+          ]);
+        });
+      });
+    }
+  }, []);
+
+  const getMangoDetails = (mangaId: string) => {
+    // MangaApis.getMangaDetails(mangaId).then((response) => {
```
Comment (CharelMarais) [inline src/components/mangaCard.tsx:57]: Make sure to remove any commented code

**Review by CharelMarais — COMMENTED** [review]
Please check the comments and make adjustments if not being worked on in a separate branch.

**Review by sechaba-mache — CHANGES_REQUESTED** [review]
Remove commented code and rename component T as mentioned by Charel

**Pull request merged**